lib/acts_as_rdfable/acts_as_rdfable_core.rb

```diff
@@ -6,11 +6,11 @@ def acts_as_rdfable(&configuration_block)
       raise InvalidClassError unless self < ActiveRecord::Base
 
       define_method :rdf_annotations do
-        RdfAnnotation.for_table(self.table_name)
+        RdfAnnotation.for_table(self.class.table_name)
```

Comment:
Sorry you had to waste your time tracking down this broken bit on master!

Rather than waste your time waiting for me to merge a bunch of changes I've got in another branch, and locally, can I just suggest that instead of leaving this as an instance method and looking up the instance's class, we solve this with:


```ruby
define_singleton_method :rdf_annotations do
         RdfAnnotation.for_table(self.table_name)
```

instead? The rationale for building out the API this way locally for me was that the RDF is really a property of the class, and not going to vary by instance, so it makes more sense to call this as `Thesis.rdf_annotations` rather than `some_instance.rdf_annotations`, particularly when I don't always have an instance, and don't want to have to dummy one up just to get access to this method. It also makes it easier to cache the lookup if it's performance critical, since we don't have to worry about a memoized copy per-instance.

If you still think the latter is more convenient, we can add a couple of convenience methods like

```ruby
define_method :rdf_annotations do
         self.class.rdf_annotations
```
as well

Reply:
No worries, everything is a learning experience! The singleton method works for me :D 

I am currently using this gem working with the AIP API, could you do a feature freeze branch under the ualbertalib account so I can use it for development and testing ? 

Thanks !

Reply:
Sure let's tag a 0.2.1 as soon as your changes are in, so you can point directly at that